Upstream patch to install Python scripts to /usr/share/clfft and not to /usr/bin.
Remove for clfft-9999 as soon as develop branch is merged into master.

From 37b1f443e516f0a87c4a728cac8291148bb31b72 Mon Sep 17 00:00:00 2001
From: Marius Brehler <marius.brehler@tu-dortmund.de>
Date: Wed, 18 Feb 2015 08:59:06 +0100
Subject: [PATCH] Install scripts/perf to share/clFFT on non WIN32
 systems

---
 src/scripts/perf/CMakeLists.txt | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/src/scripts/perf/CMakeLists.txt b/src/scripts/perf/CMakeLists.txt
index 3733789..ca45b92 100644
--- a/src/scripts/perf/CMakeLists.txt
+++ b/src/scripts/perf/CMakeLists.txt
@@ -20,5 +20,8 @@ set(GRAPHING_SCRIPTS 	measurePerformance.py
 						errorHandler.py 
 						performanceUtility.py
 						)
-
-install( FILES ${GRAPHING_SCRIPTS} DESTINATION bin${SUFFIX_BIN} )
\ No newline at end of file
+if( WIN32 )
+	install( FILES ${GRAPHING_SCRIPTS} DESTINATION bin${SUFFIX_BIN} )
+else ( )
+	install( FILES ${GRAPHING_SCRIPTS} DESTINATION share/clFFT )
+endif( )
\ No newline at end of file
-- 
2.0.4

